nixos/doc/manual/release-notes/rl-2311.section.md

```diff
@@ -43,3 +43,5 @@
 - A new option was added to the virtualisation module that enables specifying explicitly named network interfaces in QEMU VMs. The existing `virtualisation.vlans` is still supported for cases where the name of the network interface is irrelevant.
 
 - `services.nginx` gained a `defaultListen` option at server-level with support for PROXY protocol listeners, also `proxyProtocol` is now exposed in `services.nginx.virtualHosts.<name>.listen` option. It is now possible to run PROXY listeners and non-PROXY listeners at a server-level, see [#213510](https://github.com/NixOS/nixpkgs/pull/213510/) for more details.
+
+- `services.kanidm.enablePam` will now also configure PAM services to use Kanidm's PAM module.
```

Comment:
I think we can backport that as a bugfix, since we did in fact advertise PAM support as you stated.

Reply:
Means the release notes update makes no sense.

Reply:
Removed the release notes entry.